_PlatformOS/getting-started/pages/adding-visuals-pages-liquid.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Adding Visuals to Pages: Liquid
+permalink: /get-started/pages/adding-visuals-pages-liquid
+---
+This guide will help you add visuals (images) to your pages using a Liquid page with the format CSS.  
+
+## Requirements
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your code base, understand the concepts of pages, layouts, and assets, and know how to create a page.  
+
+* [Required Directory Structure]()
+* [Pages]()
+* [Layouts]()
+* [Assets]()
+* [Creating a Page]()
+
+## Steps 
+
+Adding visuals to your pages using Liquid is a three-step process:
+
+1. Prepare an image
+2. Create page `logo.css.liquid`
+3. Link to the `logo.css.liquid` page in your layout
+
+### Step 1: Prepare an image
+Put an image into the `marketplace_builder/assets/images` directory, e.g. logo.svg. 
+
+### Step 2: Create page `logo.css.liquid`
+
+Create a page with the format CSS in your `marketplace_builder/pages` directory named `logo.css.liquid`: 
+
+{% raw %}
+
+```liquid
+---
+slug: logo
+format: css
```

Comment:
I think we should promote using filename with format instead of `format: css` in yml - those should be equivalent and if they not, there is a bug. But we shouldnt promote both solutions at the same time.